source/helpers/CaseDataConverter.js

```diff
@@ -45,9 +45,11 @@ export const convertAnswersToArray = (data, formQuestions) => {
 
   Object.entries(data).forEach((answer) => {
     const [fieldId, value] = answer;
-    const { id, type, tags, ...other } = formQuestions.find((element) => element.id === fieldId);
+    const { id, type, tags, disableValueStorage, ...other } = formQuestions.find(
```

Comment:
Where does `disableValueStorage` come from? I can't find any references to it.

Reply:
disableValueStorage should be created by form builder (as a question) and be used to disable storing values for a corresponding question field/component.
Form builder disableValueStorage key values: https://github.com/helsingborg-stad/mitt-helsingborg-form-builder/commit/3d104a34af6e396f56ec0db0f11a16c2979c1d6e

Reply:
Ah I see; my form-builder was out of date 🙂.